js/desktopCapturer/screen-picker.html

```diff
@@ -0,0 +1,239 @@
+            height: 150px;
+            margin: 5px;
+            position: relative;
+            width: 40%;
+            border: 1px solid rgba(0, 0, 0, .1);
+        }
+
+        .item-container:hover {
+            box-shadow: 0 1px 5px rgba(0, 0, 0, .1), 0 5px 5px rgba(0, 0, 0, .1);
+        }
+
+        .screen-section-box {
+            -webkit-transition: all 1.0s ease;
+            width: 100%;
+            height: 100px;
+            overflow: hidden;
+            margin: 10px auto;
+        }
+
+        .img-wrapper {
+            max-height: 200px;
+            width: auto;
+        }
+
+        .screen-source-title {
+            white-space: nowrap;
+            width: 70%;
+            overflow: hidden;
+            font-size: 0.8rem;
+            color: grey;
+            margin: auto;
+            text-overflow: ellipsis;
+        }
+
+        .selected {
+            border-color: #3DA2FD;
+            border-bottom-style: solid;
+        }
+
+        footer {
+            padding: 10px 16px;
+            border-top: 1px solid rgba(0, 0, 0, 0.10);
+            display: flex;
+            align-items: center;
+            justify-content: flex-end;
+        }
+
+        button {
+            box-shadow: none;
+            border: none;
+            border-radius: 20px;
+            font-size: 0.8rem;
+            text-align: center;
+            padding: 8px 32px;
+            margin: 8px 8px 8px 0;
+            outline: none;
+            display: inline-block;
+            text-decoration: none;
+            line-height: 12px;
+        }
+
+        button:focus {
+            outline: 0;
+        }
+
+        button:active {
+            outline: 0;
+        }
+
+        .cancel-button {
+            color: rgba(0, 0, 0, 0.38);
+            background-color: rgba(255, 255, 255, 0);
+            text-transform: uppercase;
+            margin: 0;
+            padding: 8px 16px;
+        }
+
+        .cancel-button:hover {
+            background-color: rgba(255, 255, 255, 0);
+            color: black;
+        }
+
+        .share-button {
+            background-color: #3DA2FD;
+            color: #FFF;
+            cursor: pointer;
+            text-transform: uppercase;
+            box-shadow: none;
+        }
+
+        .share-button-disable {
+            background-color: rgba(221, 221, 221, 1);
+            color: rgba(148, 148, 148, 1);
+            text-transform: uppercase;
+            cursor: default;
+            box-shadow: none;
+        }
+
+        .hidden {
+            display: none;
+        }
+
+        #error-content {
+            display: none;
+            padding: 180px 0 180px;
+            border-top: 1px solid rgba(221, 221, 221, 1);
+        }
+
+        #error-content span {
+            color: rgba(0, 0, 0, 0.38);
+        }
+
+    </style>
+</head>
+<body>
+<div class="content">
+    <div class="custom-window-title">
+        <span>Choose what you'd like to share</span>
```

Comment:
Try to use HTML unicodes for symbols - https://www.fileformat.info/info/unicode/char/27/index.htm